app/konnect/org-management/sso.md

```diff
@@ -38,7 +35,7 @@ using their IdP credentials, without needing a separate login.
     * The path can be any alphanumeric string.
     * The path does not require a slash (`/`).
 
-1. Click **Save**.
+1. After clicking Save, configure the SP Entity ID and Login URL on your SAML IdP.
```

Comment:
Looks like this modification should have been done on the 'Click Save' instruction for SAML, not OIDC.